scripts/roles.json

```diff
@@ -4009,20 +4401,23 @@
     "requiredContextRole": ["group", "menu", "menubar"],
     "requiredOwnedElements": [],
     "requiredProps": ["aria-checked"],
-    "superClass": ["menuitemcheckbox", "radio"]
```

Comment:
The `radio` superclass is still listed: https://www.w3.org/TR/wai-aria-1.2/#menuitemradio

Reply:
Not in the latest editor's draft: https://rawgit.com/w3c/aria/1d856c402eafbe7823021ff4cb5ca44dc1a9edf7/#menuitemradio